src/c++/perf_analyzer/genai-perf/tests/test_console_exporter.py

```diff
@@ -24,39 +24,78 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from genai_perf import parser
 from genai_perf.export_data.console_exporter import ConsoleExporter
 from genai_perf.export_data.exporter_config import ExporterConfig
-from genai_perf.metrics import LLMMetrics, Statistics
+from genai_perf.metrics import LLMMetrics, Metrics, Statistics
 
 
 class TestConsoleExporter:
 
-    def test_pretty_print_output(self, capsys) -> None:
+    def test_streaming_llm_output(self, monkeypatch, capsys) -> None:
+        argv = [
+            "genai-perf",
+            "-m",
+            "model_name",
+            "--service-kind",
+            "openai",
+            "--endpoint-type",
+            "chat",
+            "--streaming",
+        ]
+        monkeypatch.setattr("sys.argv", argv)
+        args, _ = parser.parse_args()
+
+        metrics = LLMMetrics(
+            request_throughputs=[123],
+            request_latencies=[4, 5, 6],
+            time_to_first_tokens=[7, 8, 9],
+            inter_token_latencies=[10, 11, 12],
+            output_token_throughputs=[456],
+            output_sequence_lengths=[1, 2, 3],
+            input_sequence_lengths=[5, 6, 7],
+        )
+        stats = Statistics(metrics=metrics)
+
         config = ExporterConfig()
-        config.stats = stats
+        config.stats = stats.stats_dict
+        config.metrics = stats.metrics
+        config.args = args
+
         exporter = ConsoleExporter(config)
         exporter.export()
 
         expected_content = (
-            "                             LLM Metrics                              \n"
-            "┏━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┓\n"
-            "┃                Statistic ┃  avg ┃  min ┃  max ┃  p99 ┃  p90 ┃  p75 ┃\n"
-            "┡━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━┩\n"
-            "│ Time to first token (ms) │ 2.00 │ 2.00 │ 3.00 │ 2.99 │ 2.90 │ 2.75 │\n"
-            "│ Inter token latency (ms) │ 0.50 │ 0.00 │ 1.00 │ 0.99 │ 0.90 │ 0.75 │\n"
-            "│     Request latency (ms) │ 3.00 │ 3.00 │ 4.00 │ 3.99 │ 3.90 │ 3.75 │\n"
-            "│   Output sequence length │ 6.50 │ 6.00 │ 7.00 │ 6.99 │ 6.90 │ 6.75 │\n"
-            "│    Input sequence length │ 7.50 │ 7.00 │ 8.00 │ 7.99 │ 7.90 │ 7.75 │\n"
-            "└──────────────────────────┴──────┴──────┴──────┴──────┴──────┴──────┘\n"
-            "Output token throughput (per sec): 123.00\n"
-            "Request throughput (per sec): 456.00\n"
+            "                                LLM Metrics                                 \n"
+            "┏━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━┳━━━━━━━┳━━━━━━━┳━━━━━━━┳━━━━━━━┳━━━━━━━┓\n"
+            "┃                Statistic ┃   avg ┃   min ┃   max ┃   p99 ┃   p90 ┃   p75 ┃\n"
+            "┡━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━╇━━━━━━━╇━━━━━━━╇━━━━━━━╇━━━━━━━╇━━━━━━━┩\n"
+            "│     Request latency (ms) │  5.00 │  4.00 │  6.00 │  5.98 │  5.80 │  5.50 │\n"
```

Comment:
The order of the table rows should not change right now.
That has the potential to be a breaking change or at least not intuitive.

Reply:
I agree this could cause a breaking change since some users might be parsing the table. But I'm curious what is unintuitive about Request latency being on top of the table? I think it's more consistent since TTFT, ITL, ISL, OSL are all LLM specific metrics and so it makes sense to add them _after_ the base metrics (in our case request latency). This is consistent with the embedding console output which only has request latency in the table.

Reply:
I think you are right ultimately @nv-hwoo.
But I want to separate those look and feel changes into a separate PR.
I also want to have the chance to inform internal customers.

Reply:
Yes it's not too hard to revert the changes. Hardcoded the order to maintain the order for now. I have added ticket for this as well.